src/test/java/org/recap/ils/model/nypl/patron/_84UT.java

```diff
@@ -0,0 +1,25 @@
+package org.recap.ils.model.nypl.patron;
+
+import org.junit.Test;
+import org.recap.BaseTestCase;
+
+import static org.junit.Assert.assertTrue;
+
+public class _84UT extends BaseTestCase {
+
+    _84 inst;
+
+    @Test
+    public void test_84() {
```

Comment:
I see the below code chunk is being duplicated a lot, except that instance. Create a base class for _84 like, and have this common changes moved there